lib/stat_tracker.rb

```diff
@@ -37,8 +37,37 @@ def percentage_ties
     @games.values_at(:away_goals, :home_goals).each do |game|
       ties += 1 if game[0] == game[1]
     end
-
     ((ties/total_games)*100).round(1)
   end
 
+  def percentage_home_wins
+    percentage = (home_wins/home_games) * 100
+    p percentage
+  end
```

Comment:
Looks good very clean with the helper methods!!

Reply:
I prefer to have methods return values. If I want to print the return value, I should do that where I call the method.